packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -18,7 +18,7 @@ const {InfoTip, PropCheckBox, TexButtons} = components;
 const {getDependencies} = Dependencies;
 
 // An answer can be considered correct, wrong, or ungraded.
-const CONSIDERED = ["correct", "wrong", "ungraded"];
+const CONSIDERED = ["correct", "wrong", "ungraded"] as const;
```

Comment:
Adding `as const` means we can safely use `ReadonlyArray` below and also use the intrinsic `arr.indexOf` instead of using underscore.